docs/docs/connectors/twilio.mdx

```diff
@@ -59,3 +59,61 @@ twilio:
 
 Restart your Rasa server
 to make the new channel endpoint available for Twilio to send messages to.
+
+
+#### Receiving location data via twilio (whatsapp)
+
+You can receive location data from user from this channel you need to do the following.
+
+1. Make an intent name locationData and two entities/slots one for Latitude and one for Longitude respectively.
+
+```yaml-rasa title="domain.yml"
+intents:
+  - locationData
+slots:
+   Latitude:
+     type: text
+     mappings:
+     - type: custom
+
+   Longitude:
+     type: text
+     mappings:
+     - type: custom
```

Comment:
```suggestion
slots:
   Latitude:
     type: text
     mappings:
     - type: from_entity
       entity: Latitude

   Longitude:
     type: text
     mappings:
     - type: from_entity
       entity: Longitude
```

You can also set the slots with the entity extraction and avoid using the Custom Action altogether